### PR TITLE
Add grafana.com in domain allowing list

### PR DESCRIPTION
**Pull request by pipo02mix**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/reference/domain-allowlist/index.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ Below is a list of the external domains we require access to for our clusters to
     - Opsgenie's API is used to send alerts.
 - `gitlab.com`
     - Grafana may download plugins from repositories hosted by Gitlab.
+- `grafana.com`
+    - Grafana download a plugin directly from its API.
```
Comment (Oshratn) [inline src/content/reference/domain-allowlist/index.md:77]:
```suggestion
    - Grafana downloads a plugin directly from its API.
```

**Review by Oshratn — APPROVED** [review]
LGTM

File: src/content/reference/domain-allowlist/index.md [hunk 2]
```diff
@@ -73,6 +73,8 @@ Below is a list of the external domains we require access to for our clusters to
     - Opsgenie's API is used to send alerts.
 - `gitlab.com`
     - Grafana may download plugins from repositories hosted by Gitlab.
+- `grafana.com`
+    - Grafana downloads a plugin directly from its API.
```
Comment (JosephSalisbury) [inline src/content/reference/domain-allowlist/index.md:77]: I don't follow - which API?

Comment (glitchcrab) [reply src/content/reference/domain-allowlist/index.md:77]: it's not, it's to pull the macropower plugin from their plugin registry

**Pull request merged**